app/components/Chart/ChartTypes/NVD3BaseChart/index.js

```diff
@@ -59,6 +62,21 @@ class BaseChart extends Component {
     }
   }
 
+  /**
+   * Create tick formatter function from JSON data
+   *
+   * @param object options Options object
+   * @param string key 'xAxis' or 'yAxis'
+   */
+  _setTickFormat(options, key) {
+    if (options.tickFormatBuilder && options.tickFormatBuilder[key]) {
+      options[key] = update(options[key] || {}, { $merge: { // eslint-disable-line no-param-reassign
```

Comment:
I haven't seen the dollar sign object property prefix before like `$merge` - does that convey special meaning?